### PR TITLE
Update Script code to add column for maintainers email

### LINKED ISSUE #77: Add one more column for maintainers name and email in the script
We are trying to add two more columns for package maintainers name and email-id  in the output file of the script so that it would be easier for us to contact them and also to keep a record for us.

### PR DESCRIPTION
**Pull request by sonali8434**
Close #77  I have Updated a code bit to add one more column for maintainers name and email and also try to reduced some of the execution time of code @jwokaty Please Review.

### CODE REVIEW

**Comment by sonali8434** [comment]
I think we can also remove

```
pckgs<- c()
for (i in packageNames) {
  if (i %in% biocPckgs$Package)
    pckgs <- append(pckgs, i)
}
```
Since I have'nt use it anywhere just need you confirmation before removing this? I think this is the reason why the execution time is more although I am not sure about this.

File: inst/script/categorize_vignettes.R [hunk 1]
```diff
@@ -15,30 +15,25 @@
...
 
 # Get packages in specified Bioconductor version
-biocPckgs <- biocPkgList(
-  version,
-  repo,
-  addBiocViewParents = TRUE
-)
+biocPckgs <- biocPkgList(version, repo, addBiocViewParents = TRUE)
 
 # Create list of packages from vignette list in specified version
```
Comment (jwokaty) [inline inst/script/categorize_vignettes.R:38]: I think lines 38-44 can be removed.

**Review by jwokaty — CHANGES_REQUESTED** [review]
@sonali8434 Thanks for this PR. Could I ask you to make these two changes before I merge the PR?

File: inst/script/categorize_vignettes.R (hunk 1 above)
Comment (sonali8434) [reply inst/script/categorize_vignettes.R]: Done.

**Pull request merged**